doc/examples/filters/plot_blur_effect.py

```diff
@@ -0,0 +1,83 @@
+"""
+=========================
+Estimate strength of blur
+=========================
+
+This example shows how the metric implemented in ``filters.blur_effect``
+behaves, both as a function of the strength of blur and of the size of the
+re-blurring filter. This no-reference perceptual blur metric is described in
+[1]_.
+
+.. [1] Frederique Crete, Thierry Dolmiere, Patricia Ladret, and Marina
+   Nicolas "The blur effect: perception and estimation with a new
+   no-reference perceptual blur metric" Proc. SPIE 6492, Human Vision and
+   Electronic Imaging XII, 64920I (2007)
+   https://hal.archives-ouvertes.fr/hal-00232709
+   :DOI:`10.1117/12.702790`
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import scipy.ndimage as ndi
+
+import plotly
+import plotly.express as px
+from skimage import (
+    data, color, filters
+)
+
+
+#####################################################################
+# Generate series of increasingly blurred images
+# ==============================================
+# Let us load an image available through scikit-image’s data registry. The
+# blur metric applies to single-channel images.
+
+image = data.astronaut()
+image = color.rgb2gray(image)
+
+#####################################################################
+# Let us blur this image with a series of uniform filters of increasing size.
+
+N = 32
+blurred_images = [ndi.uniform_filter(image, size=k) for k in range(2, N, 2)]
```

Comment:
```suggestion
blurred_images = [ndi.uniform_filter(image, size=k) for k in range(2, 32, 2)]
```
`N` is not really a good parameter name and it is only used here, I think that we can avoid it ;)

Reply:
Good catch! Thanks.